modules/library/xml/src/test/java/org/geotools/xml/resolver/SchemaCacheOnlineTest.java

```diff
@@ -119,16 +121,16 @@ public void downloadHttps() throws Exception {
 
     /** Basic sanity checks of schema and test store to disk. */
     private void check(byte[] bytes) {
-        Assert.assertTrue(bytes.length > 0);
+        assertTrue(bytes.length > 0);
         String text = new String(bytes);
-        Assert.assertTrue(text.contains("GeoSciML"));
-        Assert.assertTrue(text.contains("<schema"));
-        Assert.assertTrue(text.contains("</schema>"));
+        assertTrue(text.contains("GeoSciML"));
+        assertTrue(text.contains("<schema"));
+        assertTrue(text.contains("</schema>"));
```

Comment:
Looks like these are unrelated changes done by the IDE.

Reply:
fixed